src/main/java/org/broadinstitute/hellbender/engine/filters/LibraryReadFilter.java

```diff
@@ -16,7 +16,7 @@
 public final class LibraryReadFilter extends ReadFilter implements Serializable {
     private static final long serialVersionUID = 1L;
 
-    @Argument(fullName = "library", shortName = "library", doc="The name of the library to keep", optional=false)
+    @Argument(fullName = "library", shortName = "library", doc="The name of the library to keep.", optional=false)
```

Comment:
remove period

Can also drop "The" for brevity. Would be good to clarify whether you can specify multiple libraries to keep (and if so how), and whether you can exclude a single library. This would be for cases when there are 24 libraries and I want to exclude just one; I don't want to have the list the 23 others as "keep".